package.json

```
@@ -55,7 +56,7 @@
   "license": "MIT",
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "npx pretty-quick --staged"
```

Taking @tenthirtyone's recommendation to use pretty-quick's docs over husky's:

```suggestion
      "pre-commit": "pretty-quick --staged"
```